spec/game_spec.rb

```diff
@@ -49,5 +49,12 @@
             expect(@game_1.venue).to eq('Toyota Stadium')
         end
     end
+
+    describe 'total_goals' do
```

Comment:
I like that we're using helper methods. This strategy should be implemented across the board for more of our CEO methods.